bootstrap/pkg/kfapp/minikube/minikube.go

```diff
@@ -60,56 +61,58 @@ func (minikube *Minikube) Delete(resources kftypes.ResourceEnum) error {
 }
 
 func (minikube *Minikube) generate() error {
```

Comment:
@lluunn @gabrielwen can one of you just file a separate PR to delete the minikube code?

Reply:
will do